binders/difido-testng/src/main/java/il/co/topq/difido/config/AbstractPropertiesConfigFile.java

```diff
@@ -79,22 +54,34 @@ private void initDefaultProperties() {
 			log.warning("Failed to write Difido properties to file");
 		}
 	}
-
-	public String getPropertyAsString(RemoteDifidoOptions option) {
```

Comment:
@ronyb if you already have this project, do you mind add my changes? I hope it's not too much :)